include/jwt-cpp/jwt.h

```diff
@@ -916,6 +921,110 @@ namespace jwt {
 			const size_t signature_length;
 		};
 
+#ifndef OPENSSL110
```

Comment:
can you share the release notes where this was introduced to openssl?

Reply:
https://www.openssl.org/news/openssl-1.1.1-notes.html
> Major changes between OpenSSL 1.1.0i and OpenSSL 1.1.1 [11 Sep 2018]

The one-shot signing and verification functions that EdDSA requires are each new to the 1.1.1 interface, hence the full ifdefs.